### PR TITLE
Adding a navbar component to the page  

### PR DESCRIPTION
**Pull request by NasriHaithem**
Creating a responsive navbar for our SPA

Screenshot from 2020-12-22 19-07-52
Screenshot from 2020-12-22 19-08-25

### CODE REVIEW

File: src/app/components/header/header.component.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  styleUrls: ['./header.component.css']
+})
+export class HeaderComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+  showNavItems () {
+    var x = document.getElementById("nav-items-id");
```
Comment (zguesmi) [inline src/app/components/header/header.component.ts]:
```suggestion
    let x = document.getElementById("nav-items-id");
```

**Review by zguesmi — CHANGES_REQUESTED** [review]
Bravo @NasriHaithem ! :clap:

File: src/app/components/header/header.component.ts (hunk 1 above)
Comment (NasriHaithem) [reply src/app/components/header/header.component.ts]: done , 
I added the logo too

**Review by zguesmi — APPROVED** [review]
Well done!

**Pull request merged**